r/R/examples/stat_large.R

```diff
@@ -0,0 +1,35 @@
+#library(h2owave)
+
+
+captions <- list("Our Constitution begins with those three simple words."
```

Comment:
Let's see if we can make this example a bit more understandable (and improve over the Python example).

**Improvement 1**

This example is odd to read: 
- The captions are unrelated to mtcars.
- The percentage displayed is a random value.
- The primary value originates from mtcars.

To the reader, it takes more work to read through the example and distill it down to its essence. To simplify this, either use random data for all three attributes, or use mtcars for all three attributes.

**Improvement 2**

Depending on what information is being displayed, change `foo` and `bar` to the names of the variables (e.g. 'mpg', 'efficiency', 'readout') so that it's clear which variable is being mapped to which part of the widget.

Also rename `card_value`, `card_pc`, etc. to reflect the actual data being displayed ('mpg', 'efficiency', etc.).

**Improvement 3**

The indentation looks wrong (e.g. the `$add.card(` block is indented to col 72)

**Improvement 4**

Rename the card `test` to something that reflects the data being displayed (e.g. `mtcars_stats`) so that the `$page$set("test", ...` is more understandable.